mrp_production_capacity/models/mrp.py

```diff
@@ -38,10 +24,14 @@ class MrpRoutingWorkcenter(models.Model):
 class MrpProduction(models.Model):
     _inherit = 'mrp.production'
 
+    show_split_button = fields.Boolean('Show split button', default=False)
```

Comment:
OK. Entendido. Entonces también tendré que tocar el botón de confirmar. Ya que si me crean lo OF por código, y luego la confirman, estarían confirmando una OF con cantidad superior al maximo por ciclo.